docs/user-guide/create-page.md

```diff
@@ -0,0 +1,106 @@
+---
+sidebar-position: 3
+---
+
+# Create page
+
+This is where the actual magic happens and where you cook your resume with our
+drag-and-drop easy-to-use technology, along with multiple customizations.
+
+> The create panel is the place where you get to edit the details inside your resume.
+
+It gives you **editing options on the left** and an **A4 size preview on the right**.
+
+![Create Page](/doc_img/create-page.png)
+
+## Customizations
+
+### Basic Info
+
+Here you have to give your basic information like your **full name** and **Job-Title** which will
+appear on the top of your resume.
+
+You can also add a professional headshot(under resume photo) if you like and it will appear
+on the top right corner with a circle mask.
+
+:::info
+Right now the resume photo only support PNG/JPEG image format with max size of 512KB.
+:::
+
+### Where can people contact you
+
+You can add various **links identifier of your contacts** from various website or phone number.
+
+![Contact information](/doc_img/contact-illus.png)
+
+We provide the respective icon for the following websites :
+
+- Twitter
+- Github
+- Gitlab
+- Email
+- LinkedIn
+- Phone
+- Portfolio
+- Behance
+- Dribble
+- Youtube
+- Custom
+
+:::note
+For the custom link option you have to provide the link and identifier which will appear on your resume.
+You can use these custom links to add your Leetcode, CodeChef etc.
+:::
+
+![Link adding](/gifs/add_link.gif)
+
+You only need to put the handle of the social site you're adding.
+
+### Working with a section
+
+The various sections are made with almost the same data input layout to familiarize
+the user with Resuminator and make it easy to use.
+
+- The description sub-section supports Rich-text editing and markdown.
```

Comment:
Let's link some good reference to RTE and markdown

Reply:
We are not really giving many options to the user in the RTE(only bold, italics, bullet and numbered list, left align, justify)
so is there any need for a reference to RTE?
if yes, then can you please refer any one, as I'm out of ideas on which to link

**I have already done the markdown reference link.**

Reply:
I'm thinking of making a table of markdown and its RTE shortcut
is that okay?

Reply:
Yes that works